app/presenters/content_item/linkable.rb

```diff
@@ -3,7 +3,7 @@ module Linkable
     include ActionView::Helpers::UrlHelper
 
     def from
-      organisations_ordered_by_importance + links_group(%w{worldwide_organisations ministers speaker})
+      organisations_ordered_by_importance(labelledby: 'metadata-from') + links_group(%w{worldwide_organisations ministers speaker}, labelledby: 'metadata-from')
```

Comment:
Rather than put the `labelledby` parameter down through all these methods, would it make more sense for the original methods to return an array of link titles/paths that we can iterate over at a higher level?

The hardcoded HTML ID in this helper is a long way from where that ID is defined